test_unstructured/test_utils.py

```diff
@@ -380,3 +381,9 @@ def test_calculate_shared_ngram_percentage_returns_null_vals_for_empty_str():
     percent, common_ngrams = utils.calculate_shared_ngram_percentage(str1, str2, n)
     assert percent == 0
     assert not bool(common_ngrams)
+
+
+def assign_hash_ids(elements: List[Element]) -> List[Element]:
```

Comment:
This needs to go in `unit_utils.py`, not `test_utils.py`. `test_utils` is the test module for the `unstructured.utils` module. Unit-test utilities go in `unit_utils`.

Reply:
good catch, I moved it there